app/controllers/posts_controller.rb

```diff
@@ -5,11 +5,9 @@ def index
   end
 
```

Comment:
- Hi, I would like you solves N+1 problem when fetching all posts and their comments for a user.
Example in the posts#index actions you should use `@posts = @user.posts.includes(:comments)` instead of `@posts = @user.posts`